.github/ISSUE_TEMPLATE

```diff
@@ -1,16 +1,17 @@
 For configuration or updating questions please read the README and UPGRADE documentation,
 or visit: https://gitter.im/PHP-CS-Fixer
 
-When reporting an issue (bug) please provide the following information:
+When reporting an issue (bug) please provide the following informations:
 
 The PHP version you are using:
 $ php -v
 
 PHP CS Fixer version you are using:
 $ php-cs-fixer -V
 
-The command you use to run the fixer (and the configuration file you are using if any).
+The command you use to run PHP CS Fixer (and the configuration file you are using if any).
 
-A minimal sample of valid PHP code that is not fixed correctly (if applicable).
-
-The fixed version of that code after you run the fixer and the version you expected.
+Minimal samples of PHP code:
+ * before running PHP CS Fixer (no changes)
+ * with unexpected changes applied when running PHP CS Fixer
+ * with the changes you expected instead
```

Comment:
I would put it more like a template than article.

> > The PHP version you are using:
> > $ php -v
> 
> ...................... // other questions here
> 
> > Minimal sample of PHP code before running PHP CS Fixer
> 

so after filling up it would become like

> > The PHP version you are using:
> > $ php -v
> PHP 7.1.11-1+ubuntu17.04.1+deb.sury.org+1 (cli) 
> 
> ...................... // other questions here
> 
> > Minimal sample of PHP code before running PHP CS Fixer
> ```php
> <?php
> echo "1";
> ```
> 

--------------------------


with the bullet, it looks less as a form to fill up